docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -128,19 +158,339 @@ to create the rescue image. If you used the configuration example above, you wil
+
+the :command:`bconsole` command would be this:
+
+.. code-block:: bconsole
+   :caption: restore the a ReaR Rescue Image to another system
+
+   *<input>restore client=bareosclient-fd fileset=LinuxAll current fileregex=/var/lib/rear/output/rear-.* restoreclient=client_with_cd_burner-fd</input>
+
+or if your prefer:
+
+.. code-block:: bconsole
+   :caption: restore the a ReaR Rescue Image to another system and adapt the path
+
+   *<input>restore client=bareosclient-fd fileset=LinuxAll current fileregex=/var/lib/rear/output/rear-.* restoreclient=client_with_cd_burner-fd regexwhere=!/var/lib/rear/output/!/tmp/rear-rescue-images/!</input>
+
+Make sure to test this thoroughly, especially if you are using :ref:`DataEncryption`,
+as restoring to another system needs special preparations.
+
+
+.. _section-rear-create-rescue-image-regularly:
+
+Update ReaR Rescue Image Regularly
+''''''''''''''''''''''''''''''''''
+
+While a ReaR Rescue Image is quite flexible,
+it works best if it matches the current configuration of your system.
+Any configuration change (hardware, software or environment/network)
+can require an updated rescue image.
+
+This can be automatic using different methods, besides others:
+
+* periodically (cron)
+* on system restart (as a configuration change, like a new kernel version, often requires a reboot)
+* Bareos backup job
+
+Doing this via Bareos backup job has two advantages:
+
+#. the rescue image corresponds perfectly to your backup
+#. your backup report immediately tells if somethings went wrong with your backup or your restore image.
+
+However, the size of a rescue image is around 200 MB.
+Adding this to a daily incremental backup is often unnecessary.
+
+So wrapping the creation of a new rescue image by a script to limit the number of times it get regenerated often makes sense.
+Limitations could be made based on the backup level or the age of the existing rescue image,
+like implemented in the following script with the corresponding :config:option:`dir/job` configuration:
+
+.. code-block:: sh
+   :caption: :file:`/usr/local/sbin/update-rear-rescue.sh`
+
+   #!/bin/bash
+
+   set -e
+   set -u
+
+   backuplevel="${1:-}"
+
+   REAR_OUTPUT_PATH=/var/lib/rear/output/
+
+   if [ "$backuplevel" = "Full" ]; then
+      echo "backup level = $backuplevel: recreating rescue image"
+   else
+      days=7
+      current_rescue_images=( $( find "$REAR_OUTPUT_PATH" -name "rear-*" -mtime -$days ) )
+      if  [ ${#current_rescue_images[@]} -gt 0 ]; then
+         echo "SKIPPING 'rear mkrescue', as images newer than $days days exists: ${current_rescue_images[@]}"
+         exit 0
+      else
+         echo "no current rescue image found. recreating it"
+      fi
+   fi
+
+   rear mkrescue
+   exit $?
+
+
+.. code-block:: bareosconfig
+
+   Job {
+      ...
+      Run Script {
+         Runs When = Before
+         Runs On Client = yes
+         Command = "/usr/local/sbin/update-rear-rescue.sh %l"
```

Comment:
Maybe mention that the admin needs to put the file there himself, or is this distributed by Bareos automatically?

Reply:
done